src/views/Predictions/Leaderboard/components/Results/RankingCard.tsx

```diff
@@ -51,7 +51,8 @@ const getRankingColor = (rank: number) => {
 const RankingCard: React.FC<RankingCardProps> = ({ rank, user }) => {
   const { t } = useTranslation()
   const rankColor = getRankingColor(rank)
-  const profileAvatar = useGetProfileAvatar(user.id)
```

Comment:
There is not much diff between fetching profile via useGetProfileAvatar and useProfileForAddress in request aspect. Only difference is team data fetching from onchain which can be negligible.